share/wake/lib/system/job.wake

```diff
@@ -114,6 +114,31 @@ export def setPlanOnce  v p = editPlanOnce  (\_ v) p
 export def setPlanKeep  v p = editPlanKeep  (\_ v) p
 export def setPlanShare v p = editPlanShare (\_ v) p
 
+# Create a labeled plan.
+# 'command' is a String, not List String.
+export def makeLabeledShellPlan label visible command =
+  makeShellPlan command visible
+  | setPlanLabel label
+
+export def makePlan2 = makeLabeledShellPlan
```

Comment:
Actually, I think the above methods should replace `makeShellPlan`.
We should have an analogous change for `makePlan` that still takes a list of arguments instead of using a shell script.

Alternatively, we could flip it and have shell plans be the default and non-shell plans be the exception. `makePlan` and `makeNativePlan` or something.